sw/device/lib/dif/dif_i2c.h

```diff
@@ -186,6 +204,33 @@ typedef struct dif_i2c_fmt_flags {
   bool suppress_nak_irq;
 } dif_i2c_fmt_flags_t;
 
+/**
+ * The I2C Target device records the following signals with recieved data
+ */
+typedef enum dif_i2c_signal{
+  /**
+   * The associated byte was recieved with a START signal, and should be the
```

Comment:
Heh, maybe get the others too :)
```suggestion
   * The associated byte was received with a START signal, and should be the
```